pkg/e2e/e2e.go

```diff
@@ -503,8 +503,18 @@ func cleanupAfterE2E(h *helper.H) (errors []error) {
 	// Do any addon cleanup if configured
 	log.Printf("Addon cleanup: %v", viper.GetBool(config.Addons.RunCleanup))
 	if viper.GetBool(config.Addons.RunCleanup) {
+		// By default, use the existing test harnesses for cleanup
+		harnesses := strings.Split(viper.GetString(config.Addons.TestHarnesses), ",")
+		arguments := []string{"cleanup"}
+
+		// Check if cleanup harnesses exist and if so, use those instead
+		cleanupHarnesses := viper.GetString(config.Addons.CleanupHarnesses)
+		if len(cleanupHarnesses) > 0 {
+			harnesses = strings.Split(viper.GetString(config.Addons.CleanupHarnesses), ",")
```

Comment:
nit: One last nit I missed -- just reuse `cleanupHarnesses` here instead of doing the second `viper.GetString`.

Reply:
oh duh